app/assets/scss/_search-summary.scss

```diff
@@ -32,3 +32,12 @@
   @include bold-48;
   padding-right: 5px;
 }
+
+/* On the Save search page, we display the search summary in a govukInsetText,
+so we need to adjust padding to match */
+.save-search-page {
+  .search-summary {
+      padding: 15px;
+      margin-top: 0;
+  }
+}
```

Comment:
Can we use a [spacing override class](https://design-system.service.gov.uk/styles/spacing/) instead?

Reply:
I've just noticed this actually belongs to an existing ticket, so I'll park it for now and comment there.